### PR TITLE
Release 11.0 webpages (initial version).

### PR DESCRIPTION
**Pull request by vieiro**
Pending review, vote links and final download links.

### CODE REVIEW

**Comment by vieiro** [comment]
The features page can be more easily viewed at https://github.com/vieiro/incubator-netbeans-website/blob/releases/nb110/netbeans.apache.org/src/content/download/nb110/index.asciidoc 
Feel free to ask for edits/changes at will.

**Comment by geertjanw** [comment]
Wow, it's awesome! A note is needed in the Java EE section to state that explicit support for Java EE 8, i.e., templates and drop-downs and so on, are not part of Apache NetBeans 11.0, we need to have a clear specification what we need here. Also, Java EE 8 only runs on JDK 8, not on later releases and so if you are doing development with Java EE 8, it is best to run NetBeans itself on JDK 8 (otherwise, for example, you may write code that complies with JDK 10 or something, which wouldn't be runnable on GlassFish).

**Comment by geertjanw** [comment]
Improved appearance of window system icons on HiDPI/Retina monitors (for Windows and MacOS LAFs) could also be called out explicitly at the end: https://github.com/apache/incubator-netbeans/pull/859

**Comment by geertjanw** [comment]
JEP-325 support -- maybe explicitly call that 'JEP 325: Switch Expressions (Preview)'. And point out that, right now, --enable-preview does not work in Maven projects in Apache NetBeans 11.0 yet, and point to this issue which will fix it, hence JEP-325 works in Ant-based projects in Apache NetBeans 11.0 only: https://github.com/apache/incubator-netbeans/pull/1173

File: netbeans.apache.org/src/content/download/nb110/index.asciidoc [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+
+Since then he has been steadily link:https://issues.apache.org/jira/issues/?jql=project%20%3D%20NETBEANS%20AND%20component%20%3D%20%22projects%20-%20Gradle%22%20and%20status%20!%3D%20Open[working very hard] 
+to solve many issues and doing improvements.
+
+As a result you can now:
+
+- open gradle projects more efficiently, 
+- you can run gradle tasks by double clicking them on the navigator pane, 
+- Apache NetBeans gradle projects have improved unit testing and code coverage,
+- there's better support for JPA and Spring projects (with the little support we have for that),
```
Comment (lkishalmi) [inline netbeans.apache.org/src/content/download/nb110/index.asciidoc]: Gradle projects works together with NetBeans JPA and Spring Support

File: netbeans.apache.org/src/content/download/nb110/index.asciidoc [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+image::gradle-navigator.png[Navigation of gradle build script, title="Navigating the gradle build script", role="left"]
+
+Since then he has been steadily link:https://issues.apache.org/jira/issues/?jql=project%20%3D%20NETBEANS%20AND%20component%20%3D%20%22projects%20-%20Gradle%22%20and%20status%20!%3D%20Open[working very hard] 
+to solve many issues and doing improvements.
+
+As a result you can now:
+
+- open gradle projects more efficiently, 
+- you can run gradle tasks by double clicking them on the navigator pane, 
+- Apache NetBeans gradle projects have improved unit testing and code coverage,
```
Comment (lkishalmi) [inline netbeans.apache.org/src/content/download/nb110/index.asciidoc]: Gradle supported unit test frameworks (JUnit 4, 5, TestNG) works with Code Coverage with JaCoCo.

File: netbeans.apache.org/src/content/download/nb110/index.asciidoc [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+- there's better support for JPA and Spring projects (with the little support we have for that),
+- you can have custom task executions,
```
Comment (lkishalmi) [inline netbeans.apache.org/src/content/download/nb110/index.asciidoc]: Customizable task execution and IDE action binding.

File: netbeans.apache.org/src/content/download/nb110/index.asciidoc [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+- the IDE has improved output processing,
+- you can debug more easily, even single methods,
```
Comment (lkishalmi) [inline netbeans.apache.org/src/content/download/nb110/index.asciidoc]: Run, Debug and Test even single methods.

**Comment by vieiro** [comment]
HI @lkishalmi ,
I think I've added all your reviews (a little confusing this review system) . Please be so kind as to re-review the result  (more easily viewed at https://github.com/vieiro/incubator-netbeans-website/blob/releases/nb110/netbeans.apache.org/src/content/download/nb110/index.asciidoc). 

If any further changes are requested please just add a comment with content and I'll be happy to add it to that section.

Thanks,
Antonio

**Comment by ebarboni** [comment]
@vieiro @lkishalmi I was on jenkins and see that something get wrong with tls tsl don't remember exactly. With @vierio we force build at the same time it should be ok I guess now.

**Comment by vieiro** [comment]
@ebarboni Yep, there was a problem with TLS pushing to gitbox. Could be an infra issue.

We lost the build log, though. I'll try to keep some more build logs (these are quite big) to try to see why this could happen. Could be a gitbox/infra problem.

**Comment by vieiro** [comment]
@ebarboni @lkishalmi Increased build logs to keep from 2 to 6. If this happens again let's try to keep the log and see if this is some sort of problem with infra/gitbox. Thanks!

**Comment by lkishalmi** [comment]
It's out now. Sending the announcements...

**Comment by wadechandler** [comment]
The page at http://netbeans.apache.org/download/nb110/index.html says
Apache Gradle, but Gradle is not an Apache project. I will push an update
tomorrow if nobody beats me to it. On my mobile atm.

Wade

**Comment by vieiro** [comment]
Hi @wadechandler ,
Thanks for noticing!
I've used the "edit in github" button to fix it. Should be alive soon.

**Comment by wadechandler** [comment]
Woot! Thanks Antonio!

**Pull request merged**